auto-update-dataset/python/repoArchivedCheck.py

```diff
@@ -51,7 +53,8 @@ def main():
             print("time used: ", time.time() - t1, "s")
             t1 = time.time()
         cnt += 1
-        if "This repository has been archived by the owner. It is now read-only." in r.text:
+        div = r.html.find('#js-repo-pjax-container > div.flash.flash-warn.flash-full.border-top-0.text-center.text-bold.py-2', first=True)
```

Comment:
Great change here, although this is very "flaky"/unstable as the UI can change any time. :)

Reply:
Hm I'm not sure if there's a method that works if the UI changes, I would think you would have to change this conditional regardless.

Reply:
No need to change this now, but one could traverse all elements (divs) to check if any has that text `This repository has been archived by the owner`. Of course, they could still change the text. :)